pkg/kubectl/cmd/create.go

```diff
@@ -241,7 +239,8 @@ type CreateSubcommandOptions struct {
 	// StructuredGenerator is the resource generator for the object being created
 	StructuredGenerator kubectl.StructuredGenerator
 	// DryRun is true if the command should be simulated but not run against the server
-	DryRun       bool
+	DryRun bool
+	// OutputFormat will be something like "json" or "yaml".
```

Comment:
I don't think this comment is necessary, also it could be "name", not only "json" or "yaml"